src/dataService/listPaginationService.ts

```diff
@@ -540,11 +540,33 @@ export class ListPaginationService {
       'list.resolved_id',
       'readitla_b.items_extended.extended_item_id'
     );
-    if (contentType == 'VIDEO') {
-      baseQuery.where('readitla_b.items_extended.video', 1);
-    } else {
-      baseQuery.where('readitla_b.items_extended.is_article', 1);
+
+    switch (contentType) {
+      // Deprecated
+      case 'VIDEO':
+        baseQuery.where('readitla_b.items_extended.video', 1);
+        break;
+      // Deprecated
+      case 'ARTICLE':
+        baseQuery.where('readitla_b.items_extended.is_article', 1);
+        break;
+      case 'IS_EXTERNAL':
+        baseQuery.where('readitla_b.items_extended.is_article', 0);
```

Comment:
sorry if this was answered elsewhere: does this mean all videos/images are considered 'external'?

Reply:
`Item is an un-parsable page and will be opened externally` does is_external means the clients will parse it in their end? 

we already have unmarseable right? how is this different?

Reply:
`IS_EXTERNAL` means that the parser did not parse it for some reason and it sets `is_article=0` for the given item. Clients open these articles in a web view instead of the native article view